src/Builder/SplitItemBuilder.php

```diff
@@ -42,10 +44,23 @@ class SplitItemBuilder
      */
     private $autoShortcutsRegex = '/\[(.)\]/';
 
+    /**
+     * @var CheckboxStyle
+     */
+    private $checkboxStyle;
+
+    /**
+     * @var RadioStyle
+     */
+    private $radioStyle;
+
     public function __construct(CliMenu $menu)
     {
         $this->menu = $menu;
         $this->splitItem = new SplitItem();
+
+        $this->checkboxStyle  = new CheckboxStyle();
```

Comment:
So this seems to be inconsistent with the way `CliMenu` and `CliMenuBuilder` work. `CliMenu` holds the style instances, not the builder. The builder proxies to them. But here the `SplitItemBuilder` contains the style instances. Can we make it similar to the `CliMenuBuilder` works?